### ISSUE TITLE
Modifying the face's mesh

### ISSUE THREAD

**Issue by seifselmi** [MarekKowalski/FaceSwap#7]
Hello @MarekKowalski,

I asked you a month ago about how to blend a portion of the face, and you responded that I need to save and modify the mesh using MeshLab, In order to do that; what is the mesh that I need to save, is it blendshapes or mesh ? And how am I supposed to save them into obj file, because I tried to open the mesh file in the candide.npz but the MeshLab couldn't open that. Can you give me a quick guide because I'm new in manipulating meshes and working with 3D softwares.

Thanks.

**Comment by MarekKowalski** [comment]
Hi,

Ok - here is a simple solution that should work:
 - download the .obj file [here](https://www.dropbox.com/s/pckatau3ze884bm/candide.obj?dl=0),
 - open it with meshlab,
 - find which faces (triangles) you want to remove,
 - get the indices of those faces using meshlab,
 - remove the entries with those indices from the mesh variable in zad2.py

Hope that helps.

Marek

**Comment by zhoushiwei** [comment]
Hi， Why I can't find the open mouth data  in the blendshape when show it (add the meanshape)

**Comment by MarekKowalski** [comment]
Hi,

I'm note sure what you mean, could you rephrase the question please?

Thanks,

Marek

**Comment by cryforyou** [comment]
how to get the face obj model? your link is outdated, i can't download it .

**Comment by MarekKowalski** [comment]
Hi, here it is:
https://www.dropbox.com/s/13grq3ffdvskc59/candide.obj?dl=0

Marek

**Comment by zhulei1228** [comment]
> * get the indices of those faces using meshlab,

How to get the indices of those faces using meshlab?I can't find this function in meshlab

**Comment by zhulei1228** [comment]
> Hi,
> 
> Ok - here is a simple solution that should work:
> 
> * download the .obj file [here](https://www.dropbox.com/s/pckatau3ze884bm/candide.obj?dl=0),
> * open it with meshlab,
> * find which faces (triangles) you want to remove,
> * get the indices of those faces using meshlab,
> * remove the entries with those indices from the mesh variable in zad2.py
> 
> Hope that helps.
> 
> Marek

How to get the indices of those faces using meshlab?I can't find this function in meshlab

**Comment by ljc19940403** [comment]
> Hi, here it is:
> https://www.dropbox.com/s/13grq3ffdvskc59/candide.obj?dl=0
> 
> Marek
Hi，Marek
